nautobot/extras/tests/test_filters.py

```diff
@@ -643,33 +542,17 @@ def test_associated_object_type(self):
         )
 
 
-class CustomFieldChoiceFilterSetTestCase(FilterTestCases.FilterTestCase):
```

Comment:
Huh interesting we have two test cases for CustomFieldChoice there.

Also notice that we are naming things inconsistently throughout the test code. Some model filter tests are named *FilterSetTestCase and some are just *TestCase

Reply:
Yeah, that's another pass I'd like to do in "free" time, is standardize all of our test case naming patterns. :-)